src/Symfony/Component/VarExporter/VarExporter.php

```diff
@@ -9,40 +9,38 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Component\Cache\Marshaller;
+namespace Symfony\Component\VarExporter;
 
-use Symfony\Component\Cache\Marshaller\PhpMarshaller\Configurator;
-use Symfony\Component\Cache\Marshaller\PhpMarshaller\Marshaller;
-use Symfony\Component\Cache\Marshaller\PhpMarshaller\Reference;
-use Symfony\Component\Cache\Marshaller\PhpMarshaller\Registry;
-use Symfony\Component\Cache\Marshaller\PhpMarshaller\Values;
+use Symfony\Component\VarExporter\Internal\Configurator;
+use Symfony\Component\VarExporter\Internal\Exporter;
+use Symfony\Component\VarExporter\Internal\Reference;
+use Symfony\Component\VarExporter\Internal\Registry;
+use Symfony\Component\VarExporter\Internal\Values;
 
 /**
  * @author Nicolas Grekas <p@tchwork.com>
```

Comment:
We usually put that at the end of the comment.

Reply:
moved down